src/step02.rb

```diff
@@ -6,6 +6,9 @@ module Step02
   # However, html and action callbacks not reusable.
 
   def self.create_dialog
+    # The content and callbacks are lost when the dialog is closed.
+    # The next time you show the dialog it's empty and broken.
```

Comment:
I think we should reword this. "lost", "broken" etc.

Also, I don't recall the content being gone, it's only the action callbacks that needs to be reinitialized. (However this dialog might rely on the callbacks to populate the content?)

How about?

```
# The callbacks needs to be reinitialized after the dialog has been closed.
```

Reply:
Oh, I thought the HTML needed to be re-added too. Changing this.

I think "reattached" nicely as it matches the method name.